### PR TITLE
Clone the newsweek website

### PR DESCRIPTION
**Pull request by descholar-ceo**
## What does it do?
It creates the clone of [the Newsweek](https://www.newsweek.com/) website

## Tasks
- [x] Project setup
    > * Create projects repository
    > * Setup linters
    > * Create necessary files (inde.html, and styles.css files)
- [x] Create the Newsweek website HTML skeleton
- [x] Style the HTML skeleton with a mobile-first approach
- [x] Make the website responsive to all screens {mobile, medium, and desktop}-sizes
- [x] Fix any linters error and polish the codes
- [x] Create a professional readme file
- [x] Implement the requested changes after review

### CODE REVIEW

**Review by trekab — CHANGES_REQUESTED** [review]
## Changes Required! :recycle: 
Hi @descholar-ceo and @mohammadumar28! This is impressive work! :clap: Your clone looks nice! You have met most of the requirements but there are a few changes you should add to make this project :ok_hand: These include:

- [x] For the navbar, make the login icon appear only on small screens. On other screen sizes, it should be the text `LOGIN`.
- [x] Reduce the space between your search input and the search icon in your navbar. This will make that section appear more like the original.
- [x] On small screens, `TOP STORY` should be the first article in the main section of the page.
- [x] The `Editor's Pick` section should have 4 articles per row on large screens. Yours has 2 articles per row.
- [x] On medium screens your `NEWSLETTER` section is missing the image on the leftmost end of the email input and sign up button.
- [x] For your CSS link tags, link your custom CSS files last. This will make it possible to customize any other linked styles. I have noticed that the font styles come after your custom styles import.

__Kindly make the necessary changes and submit another code review request.__
_Feel free to reach me on slack for any inquiries about this review._ :wink: 
__Treasure Kabareebe__
[GitHub](https://github.com/trekab) | [LinkedIn](https://www.linkedin.com/in/treasure-kabareebe/) | [Twitter](https://twitter.com/TKabareebe)

File: styles.css [hunk 1]
```diff
@@ -0,0 +1,421 @@
...
+
+.news-title a {
+  color: var(--black);
+  text-decoration: none;
+}
+
+.h3 a {
+  color: var(--black);
+  text-decoration: none;
+}
```
Comment (andresporras3423) [inline styles.css]: Group selectors with the same style definition.

File: styles.css [hunk 2]
```diff
@@ -0,0 +1,421 @@
...
+
+.cover-text {
+  padding: 5px 7px;
+  color: white;
+}
+
+.cover-link:hover {
+  color: var(--red);
+}
```
Comment (andresporras3423) [inline styles.css]: same style definition of the selector on line 70. Group selectors, or create new classes.

**Review by andresporras3423 — CHANGES_REQUESTED** [review]
### Status: Changes Required :recycle:
**Hello @descholar-ceo @mohammadumar28** 
Good job with your new submission for this project :muscle:  
your page looks really similar to the original :clap: :clap: :clap:  
Your project passes most of the requirements. There are just a few details to improve before you can go ahead:
- [x] Don't repeat yourself, in order to minimize code you can group selectors with the same style definition, or you can create new classes.
- [x]  On medium screen size the editor's pick should have only 2 columns, check out the original:



And your page:




Please request another review once you are ready! :+1: 

Happy coding :relaxed:

Oscar Andrés Russi Porras

File: styles.css (hunk 1 above)
Comment (descholar-ceo) [reply styles.css]: Implemented this

File: styles.css (hunk 2 above)
Comment (descholar-ceo) [reply styles.css]: Implemented this

**Review by elmejdki — APPROVED** [review]
# status: approved :heavy_check_mark: 
## Great job guys :100: :1st_place_medal: :clap:, thank you for making the required changes

you can now merge to master your project is approved.

## Happy Coding :tophat:

**Pull request merged**